src/app/components/views/homepage/homepage.component.ts

```diff
@@ -17,14 +18,12 @@ export class HomepageComponent implements OnInit, OnDestroy {
   private techUUID: string;
   private jobServiceSubscription: Subscription;
   private ngUnsubscribe: Subject<void> = new Subject<void>();
-  
-  constructor(private weatherService: WeatherService, private jobService: JobService) { }
 
-  ngOnInit(): void {    
-    this.techUUID = 'mw224g'; // TODO - Make part of a sort of "login" feature. Aaron is working on this I believe, possibly a sort of modal.
-    if (!this.jobService.getResults()) { // don't call the api if it already has data
+  constructor(private weatherService: WeatherService, private jobService: JobService, public cookieService: CookieService) { }
+
+  ngOnInit(): void {
+    this.techUUID = this.cookieService.get('user');
```

Comment:
add some sort of check if this data doesn't exist to route to the login page? Or maybe whatever security protection stuff we put in will make it so this isn't needed?